### DISCUSSION TITLE
SwarmUI "Creat New Presets" not work in Ubuntu22.04,,need help!

### DISCUSSION THREAD

**Discussion by superhighsky** [Stability-AI/StableSwarmUI#289] [Q&A]
In ubuntu system, swarmui cannot use the "Creat New Presets" function, and there is no response after clicking "Creat New Presets",anyone know why?

**Comment by mcmonkey4eva** [comment]
- What browser?
- Does the modal for preset config pop up?
- is there any error in your browser's debug console (hit F12)?
- Is there any error in Swarm's logs (in the terminal window, or Server -> Logs in the UI)

**Reply by superhighsky** [reply]
Thank you so much for the help provided, it's amazing to try it many times today and actually work after restarting！！

**Reply by superhighsky** [reply]
still can't work .
"[Error] Error in websocket handler: System.AggregateException: One or more errors occurred. (Object reference not set to an instance of an object.)
 ---> System.NullReferenceException: Object reference not set to an instance of an object.
   at StableSwarmUI.WebAPI.T2IAPI.RequestToParams(Session session, JObject rawInput) in /home/wdh/StableSwarmUI/src/WebAPI/T2IAPI.cs:line 93
   at StableSwarmUI.WebAPI.T2IAPI.GenT2I_Internal(Session session, ValueTuple`2 input, Action`1 output, Boolean isWS) in /home/wdh/StableSwarmUI/src/WebAPI/T2IAPI.cs:line 115
   --- End of inner exception stack trace ---"

**Reply by mcmonkey4eva** [reply]
You're somehow sending a generation with a preset that doesn't exist -- I pushed a commit to handle the error more cleanly

**Reply by superhighsky** [reply]
Thank you
